tests/test_portal.py

```diff
@@ -168,21 +171,25 @@ def test_message_sent(self):
 
         # confirm styling unicode functions
         body = message.style_message(message.body)
-        self.assertTrue(u'DOCTYPE' in body)
-        self.assertTrue(u'style' in body)
-        self.assertTrue(isinstance(body, unicode))
+        assert u'DOCTYPE' in body
+        assert u'style' in body
+        if sys.version_info[0] < 3:
+            assert isinstance(body, unicode)
+        else:
+            assert isinstance(body, str)
```

Comment:
@pbugni @ivan-c this if statement seemed to be the best way to handle this test, as unicode in py2 = str in py3, and any other solution that I could think of seemed to defeat the point of the test. Thoughts?